README.md

```diff
@@ -179,13 +179,15 @@ vis._send({'data': [trace], 'layout': layout, 'win': 'mywin'})
 ```
 
 ### Others
-- `vis.close`    : close a window by id
-- `vis.win_exists`: check if a window already exists by id
-- `vis.check_connection`: check if the server is connected
+- [`vis.close`](#visclose)    : close a window by id
+- [`vis.win_exists`](#viswinexists) : check if a window already exists by id
```

Comment:
#viswin_exists and #vischeck_connection are the correct links

Reply:
Ohh, that worked on [stackedit](https://stackedit.io/). Addressed in the commit.